cmd/mr_note.go

```diff
@@ -97,6 +112,7 @@ func mrNoteText() (string, error) {
 
 func init() {
 	mrCreateNoteCmd.Flags().StringSliceP("message", "m", []string{}, "Use the given <msg>; multiple -m are concatenated as separate paragraphs")
+	mrCreateNoteCmd.Flags().StringP("file", "f", "", "Use the given file as the message")
```

Comment:
```suggestion
	mrCreateNoteCmd.Flags().StringP("file", "F", "", "Use the given file as the message")
```
I'm leaning towards the short flag being a `-F` since that's how the comparable flag works on `git commit`, which is really the syntax being followed for a lot of these commands that.

Reply:
What are your thoughts? honestly I could go either way since `-f` is pretty common for this usecase too.

Reply:
That's a good point about git -F.  I'll change this to -F and retest.